patches/@rrweb__record@2.0.0-alpha.17.patch

```diff
@@ -1,37 +1,32 @@
 diff --git a/dist/record.js b/dist/record.js
-index 46ec389fefb698243008b39db65470dbdf0a3857..70db907755d68b08232e25e1b255a974f56f3c65 100644
+index 46ec389fefb698243008b39db65470dbdf0a3857..a18724d8b6ba43a30935daf257127fbb0c898541 100644
 --- a/dist/record.js
 +++ b/dist/record.js
-@@ -26,6 +26,19 @@ const testableMethods$1 = {
+@@ -26,6 +26,14 @@ const testableMethods$1 = {
    Element: [],
    MutationObserver: ["constructor"]
  };
-+const isFunction = (x) => typeof x === 'function';
-+const isAngularZonePatchedFunction = (x) => {
++
++const isAngularZonePresent = () => {
 +  try {
-+    if (!isFunction(x)) {
-+        return false;
-+    }
-+    const prototypeKeys = Object.getOwnPropertyNames(x.prototype || {});
-+    return prototypeKeys.some((key) => key.indexOf('__zone'));
++    return !!globalThis.Zone
 +  } catch {
-+    // we've seen some intermittent problems in Safari since introducing this check
 +    return false
 +  }
```

Comment:
turns out it is way simpler to just check if the global zone is present
if it's not, then angular didn't taint this

just like with the previous solution, folk can configure this to not be called zone, but then they can run posthog outside of the zone 🤷